.gitignore

```diff
@@ -36,3 +36,4 @@ nosetests.xml
 virtenv
 .vscode
 test_results
+.projectile
```

Comment:
Could you explain what this is please?

Reply:
This is Emacs projectile module. I started to use it recently.

Reply:
You can add this to ~/.gitignore and this will then be ignored in all your git repositories

Reply:
@pypingou This will be probably better solution, I hope it is working on the Silverblue now. Last time I tried it, it didn't worked.

Reply:
`~/.gitignore` is still ignored, but I removed it from `.gitignore`